codec/src/main/java/io/netty/handler/codec/compression/BrotliDecoder.java

```diff
@@ -0,0 +1,29 @@
+package io.netty.handler.codec.compression;
+
+import com.nixxcode.jvmbrotli.dec.BrotliInputStream;
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.ByteBufInputStream;
+import io.netty.buffer.ByteBufOutputStream;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.handler.codec.ByteToMessageDecoder;
+
+import java.util.List;
+
+public class BrotliDecoder extends ByteToMessageDecoder {
+
+    @Override
+    protected void decode(ChannelHandlerContext ctx, ByteBuf in, List<Object> out) throws Exception {
+        ByteBuf decompressed = ctx.alloc().buffer();
```

Comment:
Isn't this the compressed content?

Reply:
`decompressed` will have content of decompressed data.